rcl_action/src/rcl_action/graph.c

```diff
@@ -211,6 +223,10 @@ rcl_action_get_names_and_types(
   rcl_ret_t nat_fini_ret = rcl_names_and_types_fini(&topic_names_and_types);
   if (RCL_RET_OK != nat_fini_ret) {
     ret = rcl_names_and_types_fini(action_names_and_types);
+    if (RCL_RET_OK != ret) {
+      RCUTILS_SET_ERROR_MSG(
+        "Freeing names and types failed while handling a previous error. Leaking memory!");
```

Comment:
Missed one
```suggestion
        "Freeing names and types failed while handling a previous error. Leaking memory!\n");
```

Reply:
Arg, I'm having a rough time with this.  Thanks for pointing it out, fixed now.